plugin/plugin.go

```diff
@@ -76,6 +76,8 @@ const (
 	GCPlugin Type = "io.containerd.gc.v1"
 	// EventPlugin implements event handling
 	EventPlugin Type = "io.containerd.event.v1"
+	// LeasesPlugin implements lease manager
+	LeasesPlugin Type = "io.containerd.leases.v1"
```

Comment:
```suggestion
	LeasesPlugin Type = "io.containerd.lease.v1"
```

Other plugins (except `metadata`) do not use plural form